features/crashprobe.feature

```diff
@@ -2,7 +2,7 @@ Feature: Reporting crash events
 
 Scenario: Calling __builtin_trap()
     When I set environment variable "BUGSNAG_API_KEY" to "a35a2a72bd230ac0aa0f52715bbdc6aa"
-    And I configure the app to run on "iPhone8-11.2"
+    And I configure the app to run on "iPhone8-11.3"
```

Comment:
One of our original goals was to setup mazerunner so that it can run on multiple devices, with different models + OS versions. Would now be a good time to extract this to a variable/command line option for iOS?

Reply:
Yes. I marked it as in progress to pull the pieces around a bit. Rather than running all devices at once though, which thus far has been flaky, I'm planning to configure it so you can run it all on a particular device. That way we can also divide up the CI builds for reasonable time lengths.